src/audio/pcm_converter/pcm_converter_generic.c

```diff
@@ -29,9 +29,70 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#define BYTES_TO_U8_SAMPLES	0
 #define BYTES_TO_S16_SAMPLES	1
 #define BYTES_TO_S32_SAMPLES	2
 
+#if CONFIG_PCM_CONVERTER_FORMAT_U8 && CONFIG_PCM_CONVERTER_FORMAT_S32LE
+static int pcm_convert_u8_to_s32(const struct audio_stream __sparse_cache *source,
```

Comment:
may need add another patch to add it to hifi version as well, since current build enabled hifi3 or hifi4.

Reply:
Let's do it in a separate PR.

Reply:
you may need add it in this PR, otherwise, U8 support is not enabled fully for S32, because this part is not built with default,  just simply copy this generic C code to hifi3 or hifi4 like max function is fine.